<?xml version="1.0"?>
<launch>
		
    <node name="gripper_routing" pkg="sofar_hbc_12" type="gripper_routing" args="--wait"/> 
    
    <include file="$(find sofar_hbc_12)/launch/sofar_hbc_12.launch" />
    
</launch>
